banjo/ast-decl.hpp

```diff
@@ -6,6 +6,7 @@
 
 #include "ast-base.hpp"
 #include "specifier.hpp"
+#include "ast-def.hpp"
```

Comment:
Don't include other AST headers here. It adds too many dependencies. If you needed to add this for code below, then move that code into the .cpp file and include the header there.